src/test/java/com/diffplug/gradle/pde/PdeInstallationPropertiesTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.gradle.pde;
+
+import org.gradle.api.Project;
+import org.gradle.api.plugins.ExtraPropertiesExtension;
+import org.gradle.testfixtures.ProjectBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PdeInstallationPropertiesTest {
+
+	@Test
+	public void testUpdateSiteProperty() {
+
+		String expectedSite = "newUpdateSite";
+		String expectedId = "pdeId";
+
+		Project project = ProjectBuilder.builder().build();
+		ExtraPropertiesExtension props = project.getExtensions().getExtraProperties();
+
+		props.set("GOOMPH_PDE_VER", "1.0.0");
```

Comment:
This unit test fails on Mac because of the check in PdeInstallation constructor:

`com.diffplug.gradle.pde.PdeInstallationPropertiesTest > testUpdateSiteProperty FAILED
    java.lang.IllegalArgumentException: On mac, must be 4.5.0 (Mars) or later, because of folder layout problems.
        at com.diffplug.gradle.pde.PdeInstallation.<init>(PdeInstallation.java:122)
        at com.diffplug.gradle.pde.PdeInstallation.from(PdeInstallation.java:110)
        at com.diffplug.gradle.pde.PdeInstallation.fromProject(PdeInstallation.java:105)
        at com.diffplug.gradle.pde.PdeInstallationPropertiesTest.testUpdateSiteProperty(PdeInstallationPropertiesTest.java:39)

com.diffplug.gradle.pde.PdeInstallationPropertiesTest > testUpdateSitePropertyDownwardsCompatibilityDueTypo FAILED
    java.lang.IllegalArgumentException: On mac, must be 4.5.0 (Mars) or later, because of folder layout problems.
        at com.diffplug.gradle.pde.PdeInstallation.<init>(PdeInstallation.java:122)
        at com.diffplug.gradle.pde.PdeInstallation.from(PdeInstallation.java:110)
        at com.diffplug.gradle.pde.PdeInstallation.fromProject(PdeInstallation.java:105)
        at com.diffplug.gradle.pde.PdeInstallationPropertiesTest.testUpdateSitePropertyDownwardsCompatibilityDueTypo(PdeInstallationPropertiesTest.java:57)
`

Probably the check is wrong and the test is OK. I don't think it makes sense that GOOMPH_PDE_VER must match Eclipse version numbers?

Reply:
The check is actually important.  eclipse moved the locations of a lot of files around in eclipse 4.5.0 on mac.  Supporting pre<4.5.0 on mac is basically supporting a fourth OS.  Better to change `1.0.0` to `4.5.0` I think.

Also, the eclipse releases are looked-up manually in this: https://github.com/diffplug/goomph/blob/master/src/main/java/com/diffplug/gradle/pde/EclipseRelease.java